test/expandKeyValue_test.js

```diff
@@ -1,56 +1,48 @@
-/**
- * Copyright 2012 Google Inc. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
+goog.require('helper');
```

Comment:
Please use goog.module syntax

https://github.com/google/closure-library/wiki/goog.module:-an-ES6-module-like-alternative-to-goog.provide#googmodule

Reply:
I've updated the syntax in this PR though to actually run the test PR #26 will need to be merged first. Thank you for the suggestion!